static/app/components/onboardingWizard/onboardingCard.tsx

```diff
@@ -0,0 +1,137 @@
+    return null;
+  }
+
+  const [onboardingState] = usePersistedOnboardingState();
+  if (!onboardingState) {
+    return null;
+  }
+  // Projects selected during onboarding but not received first event
+  const projects = onboardingState.selectedPlatforms
+    .map(platform => onboardingState.platformToProjectIdMap[platform])
+    .map(projectId => allProjects.find(p => p.slug === projectId))
+    .filter(project => project && !project.firstEvent) as Project[];
+  return projects.length > 0 ? (
+    <Fragment>
+      <Heading>{t('Setup Projects')}</Heading>
+      <TaskCard key="onboarding-continue-card">
+        <OnboardingTaskProjectList>
+          {projects.slice(0, MAX_PROJECT_COUNT).map(p => (
+            <OnboardingTaskProjectListItem
+              key={p.id}
+              to={`/onboarding/${org.slug}/setup-docs/?project_id=${p.id}`}
+              onClick={() => {
+                trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                  platform: p.platform,
+                  organization: org,
+                });
+              }}
+            >
+              <OnboardingTaskProjectListItemInner>
+                <PlatformIcon platform={p.platform || 'default'} />
+                {p.slug}
+                <PulsingIndicator />
+                <PulsingIndicatorText>{t('Waiting for event')}</PulsingIndicatorText>
+                <IconChevron direction="right" />
+              </OnboardingTaskProjectListItemInner>
+            </OnboardingTaskProjectListItem>
+          ))}
+          {projects.length > MAX_PROJECT_COUNT && (
+            <OnboardingTaskProjectListItem
+              to={`/onboarding/${org.slug}/setup-docs/`}
+              onClick={() => {
+                trackAdvancedAnalyticsEvent('growth.onboarding_quick_start_cta', {
+                  organization: org,
+                });
+              }}
+            >
+              <OnboardingTaskProjectListItemInner>
+                <IconEllipsis />
+                {tct('and [num] more', {num: projects.length - MAX_PROJECT_COUNT})}
+              </OnboardingTaskProjectListItemInner>
+            </OnboardingTaskProjectListItem>
+          )}
+        </OnboardingTaskProjectList>
+      </TaskCard>
+    </Fragment>
+  ) : null;
+}
+
+const Heading = styled(motion.div)`
+  display: flex;
+  color: ${p => p.theme.purple300};
+  font-size: ${p => p.theme.fontSizeExtraSmall};
+  text-transform: uppercase;
+  font-weight: 600;
+  line-height: 1;
+  margin-top: ${space(3)};
+`;
+
+const TaskCard = styled(motion(Card))`
+  position: relative;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+`;
+
+const OnboardingTaskProjectList = styled('div')`
+  display: flex;
+  flex-direction: column;
+`;
+
+const OnboardingTaskProjectListItemInner = styled('div')`
+  padding: ${space(1)} 0;
+
+  display: flex;
+  align-items: center;
+  gap: ${space(1)};
+`;
+const OnboardingTaskProjectListItem = styled(Link)`
```

Comment:
```suggestion
const OnboardingTaskProjectListLink = styled(Link)`
```

?

Reply:
I actually like `Item` better because that's how Bootstrap name it https://getbootstrap.com/docs/4.0/components/list-group/

Reply:
But it's a link isn't it :)

Reply:
well yeah but it's also a list item

Reply:
doesn't look like a `li` element to me :trollface:

Reply:
but it also doesn't look like a link
[this](https://sentry.io) is how a link should look like

Reply:
Yeah it's fine, was just a suggestion here